.github/workflows/release.yml

```diff
@@ -143,15 +152,13 @@ jobs:
           name: pyproject
           path: sed-processor
 
-      # new_branch set for cases when tagging a release
       - name: Commit changes
+        env:
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
```

Comment:
Which permissions will this token have? Does it allow to bypass the pull request rule?

Reply:
The token has the ones you set before in the app. The only thing that's different is that I allow the app to bypass branch protection. You can see that if you check the branch protection rules